src/main/java/com/example/pawgle/like/LikeReceiver.java

```diff
@@ -0,0 +1,24 @@
+package com.example.pawgle.like;
+
+import com.example.pawgle.user.User;
+import jakarta.persistence.*;
+import lombok.NoArgsConstructor;
+
+/**
+ * 나를 호감 표시 한 사
+ */
+@Entity
+@NoArgsConstructor
+public class LikeReceiver {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "like_receiver_id")
+    private Long id;
+
+    //호감 보낸 사람
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User receiver;
```

Comment:
```suggestion
/**
 * 나를 호감 표시 한 사
 */
@Entity
@NoArgsConstructor
public class LikeReceiver {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "like_receiver_id")
    private Long id;

    //호감 보낸 사람
    @ManyToOne
    @JoinColumn(name = "sender_id")
    private User sender;
    
    // 호감 받는 사람
    @ManyToOne
    @JoinColumn(name = "receiver_id")
    private User receiver;
```
저는 호감을 조회 할 때 두개의 테이블을 join을 써서 데이터를 가져온다고 가정을 하였을 때 하나의 호감 테이블에서 데이터를 조회하는 방식보다 속도가 더 느릴거 같다는 생각과 호감 정보를 저장을 할 때 DB에 2번 커낵션이 이루어진다고 생각을 하였고, 추후 유저가 많아졌을 때 성능상으로 이슈가 되지 않을까 하여 위와 같은 식으로 보낸자와 받는자의 Entity를 하나로 하는 방법이 더 낫지 않을까 해서 코멘트 남깁니다!

Reply:
match 테이블은 유저가 하루가 지나 랜덤한 상대를 소개를 받거나 특정 조건에 의하여 소개를 받을 때 어떤 유저를 소개를 받았는지에 대해 데이터를 저장하는 용도의 테이블로 생각을 했습니다!

Reply:
추가로 @NoArgsConstructor(access = AccessLevel.PROTECTED) 해당 방식으로 적용하는것이 좀더 보안에 좋을거 같습니다!

https://velog.io/@kevin_/%EB%82%B4%EA%B0%80-NoargsConstructor-access-AccessLevel.PROTECTED%EB%A5%BC-%EC%99%9C-%EC%9E%91%EC%84%B1%ED%96%88%EC%9D%84%EA%B9%8C

관련 링크 첨부드립니다 :)

Reply:
좋은 의견 감사합니다!! 반영하겠습니다👍🏻